### DISCUSSION TITLE
Create greater choke groups

### DISCUSSION THREAD

**Discussion by ljubomirpapuga** [bangcorrupt/hacktribe#35] [Ideas]
Currently, there are only two choke groups that can be turned on and off - pads 15 and 16, and pads 17 and 18. 

These two groups also do not interact with each other when they are both on (eg. sound on pad 15 does not choke sound on pad 17).

It would be beneficial if we could have choke groups that would include a greater number of pads.

At least two choke groups - pads 1 to 8, and pads 9 to 16, or event to make all 16 pads choke each other.

This would be quite beneficial for the production of boom-bap music and sample flipping.

**Comment by bangcorrupt** [comment]
I agree.  I'd like to add freely assignable mute groups.

**Comment by ljubomirpapuga** [comment]
Thanks for a quick response, and for the all hard work you are doing. Is there any way we can get involved more?

**Reply by bangcorrupt** [reply]
No problem.  Have a look at anything labeled "good first issue".  I'll add more information on how people can help with these.

**Comment by ljubomirpapuga** [comment]
Hey, I am still interested in this? :) Were any possibilities to implement this revealed? :)

**Reply by bangcorrupt** [reply]
I haven't looked into this yet.  You can follow development activity by watching the repo or viewing the [network graph](https://github.com/bangcorrupt/hacktribe/network).
